bank.py

```diff
@@ -21,3 +22,27 @@ def get_companies(self, name):
     @classmethod
     def search_companies(cls, name, clause):
         return [('bank.party.companies',) + tuple(clause[1:])]
+
+
+class BankAccountNumber(metaclass=PoolMeta):
+    __name__ = 'bank.account.number'
+
+    @classmethod
+    def __register__(cls, module_name):
+        pool = Pool()
+        Module = pool.get('ir.module')
+
+        cursor = Transaction().connection.cursor()
+
+        party_companies = Module.search([
+            ('name', '=', 'party_company'),
+            ('state', '!=', 'activated'),
+            ], limit=1)
```

Comment:
Is this necessary? I guess you could use ALTER TABLE IF EXISTS:

https://www.postgresql.org/docs/15/sql-altertable.html